app/views/applications/new.html.erb

```diff
@@ -1,9 +1,9 @@
- 
+
  <p> Check the pet(s) you'd like to adopt below </p>
  <%= form_tag "/applications", method: :post do %>
     <% favorites.selected_pets.each do |pet| %>
     <span id= "pet_<%= pet.id %>">
-      <%= check_box_tag "pet_ids[]", pet.id %> <%= pet.name %> <br>   
+      <%= check_box_tag "pet_ids[]", pet.id %> <%= link_to pet.name, "/pets/#{pet.id}" %> <br>
```

Comment:
adding links to allow users to click on a `pet` name on every page and view its show page